tests/integration/cli/repos_test.py

```diff
@@ -0,0 +1,69 @@
+from os import path
+import shutil
+from subprocess import check_output
+
+import git
+
+from dusty.constants import REPOS_DIR
+from ...fixtures import busybox_single_app_bundle_fixture
+from ...testcases import DustyIntegrationTestCase
+
+
+class TestReposCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestReposCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=1)
+        self.set_up_fake_local_repo(path='/tmp/fake-repo-1', short_name='fake-repo-1')
+
+    def tearDown(self):
+        super(TestReposCLI, self).tearDown()
+        self.tear_down_fake_local_repo(path='/tmp/fake-repo-1')
+
+    def test_repos_list(self):
+        result = self.run_command('repos list')
+        self.assertInSameLine(result, 'Full Name', 'Short Name', 'Local Override')
+        self.assertInSameLine(result, 'github.com/gamechanger/dusty-example-specs', 'dusty-example-specs', self.overridden_specs_path)
+        self.assertInSameLine(result, 'tmp/fake-repo', 'fake-repo')
+
+    def test_repos_override(self):
+        result = self.run_command('repos list')
+        self.assertInSameLine(result, 'fake-repo', '/tmp/fake-repo')
+        self.assertNotInSameLine(result, 'fake-repo', '/tmp/fake-repo-1')
+        self.run_command('repos override fake-repo /tmp/fake-repo-1')
+        result = self.run_command('repos list')
+        self.assertInSameLine(result, 'tmp/fake-repo', 'fake-repo', '/tmp/fake-repo-1')
+
+    def test_repos_manage(self):
+        result = self.run_command('repos list')
+        self.assertNotInSameLine(result, 'fake-repo', '/tmp/fake-repo-1')
```

Comment:
You don't need to this stuff like this every time you use it. Have one test that'll catch if it's broken, then just use it freely in other places.